onchain/rollups/hardhat.config.ts

```diff
@@ -34,7 +34,7 @@ const ppath = (packageName: string, pathname: string) => {
 };
 
 const networkConfig = (chain: Chain): HttpNetworkUserConfig => {
-    let url = chain.rpcUrls.public.http.at(0);
+    let url = process.env.RPC_URL || chain.rpcUrls.public.http.at(0);
 
     // support for infura and alchemy URLs through env variables
     if (process.env.INFURA_ID && chain.rpcUrls.infura?.http) {
```

Comment:
So here if both RPC_URL and INFURA_ID are defined, it will choose the Infura URL, right?

Reply:
Yes. INFURA_ID takes priority.